pkg/plugins/generic/generic_plugin.go

```diff
@@ -141,6 +139,36 @@ func (p *GenericPlugin) OnNodeStateChange(new *sriovnetworkv1.SriovNetworkNodeSt
 	return
 }
 
+// CheckStatusChanges verify whether SriovNetworkNodeState CR status present changes on configured VFs.
+func (p *GenericPlugin) CheckStatusChanges(current *sriovnetworkv1.SriovNetworkNodeState) (bool, error) {
+	log.Log.Info("generic-plugin CheckStatusChanges()")
+
+	for _, iface := range current.Spec.Interfaces {
+		found := false
+		for _, ifaceStatus := range current.Status.Interfaces {
+			if iface.PciAddress == ifaceStatus.PciAddress && !iface.ExternallyManaged {
```

Comment:
Hi please a TODO and comment here that we skip the externallyManage with a link to the issue we open on the repo please

Reply:
done